test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -231,7 +231,8 @@ call_log_manager::call_log_query_timestamp(const json &call_log_entry)
         session_simulator *session = get_session(session_id);
 
         ret = session->query_timestamp(config, hex_ts);
-    }
+    } else
```

Comment:
nit: Our other examples of exception raising look like `throw std::invalid_argument("error_message")`

Reply:
This looks like a better approach, thanks!

Reply:
Updated in [b684e52](https://github.com/wiredtiger/wiredtiger/pull/8599/commits/b684e527bda0724e4c15f7b713e232f50babf4ef)